client/landing/jetpack-cloud/sections/pricing/index.ts

```diff
@@ -8,10 +8,7 @@ import page from 'page';
  */
 import { makeLayout, render as clientRender } from 'controller';
 import { pricing } from 'landing/jetpack-cloud/sections/pricing/controller';
-import isJetpackCloud from 'lib/jetpack/is-jetpack-cloud';
 
 export default function () {
-	if ( isJetpackCloud() ) {
```

Comment:
We don't need this check since this code will only be included in Jetpack Cloud's build process.